pkg/ctl/sender/client_docker_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/newrelic/infrastructure-agent/pkg/helpers"
 )
 
-func TestNewContainerisedClient(t *testing.T) {
+func TestNewDockerClient(t *testing.T) {
```

Comment:
It's Legacy test, I just changed the name because it's not called TestNewContainerisedClient anymore but TestNewDockerClient.
Yes you are right it's not so useful, I did the other one just doing the same as this legacy test.